doc/convert-to-man

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
-pandoc -s -t man rg.1.md -o rg.1
-sed -i 's/\.TH.*/.TH "rg" "1"/g' rg.1
+pandoc -s -t man rg.1.md -o rg.1 &&
+sed -i.bak 's/\.TH.*/.TH "rg" "1"/g' rg.1 &&
+rm -f rg.1.bak # BSD `sed` requires the creation of a back-up file
```

Comment:
I found that this script didn't run on macOS with just `sed -i`, as the BSD version of `sed` requires the back-up suffix. So i added that in a way that should be compatible with both GNU and BSD `sed`. I also added the `&&` to make the script bail if any part of the process fails.